### PR TITLE
[DO NOT MERGE] SEO edits for advanced hunting

### PR DESCRIPTION
**Pull request by jdmartinez36**
Edited metadata title and description to add keywords.

Edited body to added "cyber threat" keyword.

@DaniHalfin 

Content for this PR contains edits from PR 2155.

### CODE REVIEW

File: microsoft-365/security/mtp/advanced-hunting-overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview - Advanced hunting
-description: Learn about advanced hunting queries in Microsoft 365 and how to use them to proactively find threats and weaknesses in your network
+title: Overview - Advanced cyber threat hunting - Microsoft 365 security
```
Comment (lomayor) [inline microsoft-365/security/mtp/advanced-hunting-overview.md:2]: Capability name is "Advanced hunting" so we should not change it to "Advanced cyber threat hunting"

File: microsoft-365/security/mtp/advanced-hunting-overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+description: Learn about advanced hunting queries in Microsoft Threat Protection (MTP) and how to use them to proactively find threats and weaknesses in your network.
```
Comment (lomayor) [inline microsoft-365/security/mtp/advanced-hunting-overview.md:3]: Has use of "MTP" in public-facing content been approved?

**Review by lomayor — CHANGES_REQUESTED** [review]
Please check comments. Thanks!

**Comment by DaniHalfin** [comment]
becomes irrelevant post rebrand

**Pull request closed without merging**